CODING_STANDARD.md

```diff
@@ -188,6 +188,10 @@ Formatting is enforced using clang-format. For more information about this, see
 - Avoid `assert`. If the condition is an actual invariant, use INVARIANT,
   PRECONDITION, POSTCONDITION, CHECK_RETURN, UNREACHABLE or DATA_INVARIANT. If
   there are possible reasons why it might fail, throw an exception.
+    - Use "should" style statements for messages in invariants (e.g. "Array
```

Comment:
While at it: should those messages start with a lowercase or an uppercase character?

Reply:
72 upper case vs. 100+ lower case so going with lower case.